core/src/test/java/org/infinispan/functional/FunctionalEncoderTest.java

```diff
@@ -0,0 +1,29 @@
+package org.infinispan.functional;
+
+import org.infinispan.configuration.cache.ConfigurationBuilder;
+import org.infinispan.configuration.cache.StorageType;
+import org.testng.annotations.Test;
+
+@Test(groups = "functional", testName = "functional.FunctionalEncoderTest")
+public class FunctionalEncoderTest extends FunctionalMapTest {
```

Comment:
for the Bifunction mapper, simply because it is not used haha. I wanted to delete but I  didn't see I left it.
Ok to add tx tests and so the support. And ok to test with Advanced Caches and not just the store type.